lib/rubocop/sorbet_from_contract_service.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+module Sorbet
+  class SorbetFromContractService
+    class AutoCorrectError < StandardError; end
+    CONTRACT_PATTERN = RuboCop::NodePattern.new("$(send _ :Contract $... (:hash (:pair $_ $_)))")
+
+    EXTEND_T_SIG_PATTERN = RuboCop::NodePattern.new("(send _ :extend (const (const _ :T) :Sig))")
```

Comment:
Can't we use inheritance or composition to now repeat the same node patterns over and over?

Reply:
These were unused, removed https://github.com/Shopify/rubocop-sorbet/pull/27/files#diff-f6aa818ee249759bf642b3022cff5d25R6